python/psautohint/__main__.py

```diff
@@ -701,18 +709,12 @@ def get_options(args):
 def main(args=None):
     options = get_options(args)
 
-    if options.logFile:
-        open(options.logFile, 'w', encoding="utf-8")
-
     try:
         hintFiles(options)
     except Exception as err:
-        logMsg("\t%s" % err)
+        logging.exception("%s", err)
```

Comment:
Logging.exception already prints the traceback including the last exception error. The logged message should be something else, such as “Unhandled exception occurred”